tools/test-js-with-puppeteer

```diff
@@ -76,6 +76,12 @@ def run_tests(files: Iterable[str], external_host: str) -> None:
             ret = 1
             ret = run_tests()
     if ret != 0:
+        print("""
+The Puppeteer frontend tests failed! For any help feel free to ask in czo""", file=sys.stderr)
```

Comment:
we do not have any page in docs to help for debugging. Once we have it we should make this something like  "For help debugging, read:  https://zulip.readthedocs.io/en/latest/testing/testing-with-casper.html"